spec/controllers/blackouts_controller_spec.rb

```diff
@@ -223,23 +223,37 @@
     end
 
     context 'PUT update' do
+      shared_examples_for 'does not update blackout' do |attributes|
+        before { put :update, id: FactoryGirl.create(:blackout),
+                 blackout: attributes }
+
+        it { is_expected.to set_flash }
+        it { is_expected.to render_template(:edit) }
+        it 'should not save the blackout' do
+          expect(attributes).not_to be_nil
+          expect(assigns(:blackout)).not_to be_nil
+          expect(assigns(:blackout)[:notice]).not_to eq(attributes[:notice])
```

Comment:
this should be split up into separate tests / refactored to only have one `expect`